Mage/Command/Factory.php

```diff
@@ -43,14 +43,17 @@ public static function get($commandName, Config $config)
             // try a custom command
             $className = 'Command\\' . $commandName;
 
+            // TODO use a custom exception
             if (!class_exists($className)) {
                 throw new Exception('Command "' . $commandName . '" not found.');
             }
         }
 
         /** @var AbstractCommand $instance */
+        // TODO dependencies like $config should be injected into constructor
```

Comment:
@eps90 Does the comment of @andres-montanez in #88 really mean that no external PHP *packages* should be used? That issue is about a feature which can only be implemented using a non-standard PHP *extension* like [ncurses](http://php.net/manual/en/book.ncurses.php).

Also, does Magallanes really live as a PEAR package? It's neither mentioned in the documention nor can I see a PEAR package file (package.xml).